runtime/gc_vlhgc/GlobalMarkCardScrubber.cpp

```diff
@@ -171,6 +177,37 @@ MM_GlobalMarkCardScrubber::scrubMixedObject(MM_EnvironmentVLHGC *env, J9Object *
 	return doScrub;
 }
 
+
+void
+stackSlotIteratorForGlobalMarkCardScrubber(J9JavaVM *javaVM, J9Object **slotPtr, void *localData, J9StackWalkState *walkState, const void *stackLocation)
+{
+	StackIteratorData4GlobalMarkCardScrubber *data = (StackIteratorData4GlobalMarkCardScrubber *)localData;
+	if (*data->doScrub) {
+		*data->doScrub = data->globalMarkCardScrubber->mayScrubReference(data->env, data->fromObject, *slotPtr);
+	}
+}
+
+bool MM_GlobalMarkCardScrubber::scrubContinuationObject(MM_EnvironmentVLHGC *env, J9Object *objectPtr)
+{
+	bool doScrub = scrubMixedObject(env, objectPtr);
+
+	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
+	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr)) {
+		StackIteratorData4GlobalMarkCardScrubber localData;
+		localData.globalMarkCardScrubber = this;
+		localData.env = env;
+		localData.doScrub = &doScrub;
+		localData.fromObject = objectPtr;
+
+		bool bStackFrameClassWalkNeeded = false;
+#if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
+		bStackFrameClassWalkNeeded = true;
```

Comment:
Another tough one, but by looking here:

https://github.com/eclipse-openj9/openj9/blob/master/runtime/gc_vlhgc/GlobalMarkCardScrubber.cpp#L162

I'd say this should be always false.